static/src/javascripts/bootstraps/common.js

```diff
@@ -162,6 +164,14 @@ define([
                 navigation.init();
             },
 
+            initialiseStickyHeader: function () {
+                var ViewabilityTest = ab.getParticipations().Viewability;
+
+                if (ab.testCanBeRun('Viewability') && ViewabilityTest && ViewabilityTest.variant === 'variant' && config.page.contentType !== 'Interactive') {
```

Comment:
Dunno if this helper might be of use in shortening this

https://github.com/guardian/frontend/blob/master/static/src/javascripts/projects/common/modules/experiments/ab.js#L221-L224